helpers/result.go

```diff
@@ -0,0 +1,25 @@
+package helpers
+
+import (
+	"go-google-scraper-challenge/models"
+)
+
+func PrepareResultSet(results []*models.Result) [][]*models.Result {
+	if len(results) == 0 {
+		return nil
+	}
+
+	if len(results) > 10 {
```

Comment:
It is not a requirement, Beego has also built in [pagination](https://beego.me/docs/mvc/view/page.md) support

Reply:
@junan Yeah actually this is not in the requirement, I just want to display them in 2 tables to make use of the rest of the space on the page 😅  but I'll remove them for now and will work on this later on the [pagination task](https://github.com/carryall/go-google-scraper-challenge/issues/55)